src/Polymorph-Widgets.package/MorphicUIManager.class/instance/lowSpaceWatcherDefaultAction..st

```diff
@@ -19,6 +19,8 @@ lowSpaceWatcherDefaultAction: theInterruptedProcess
 		newDebugSessionNamed: 'Space is low' 
 		startedAt: preemptedProcess suspendedContext.		
 	debugSession logStackToFileIfNeeded.
+	debugSession isAboutUIProcess ifTrue: [ 
+		DefaultExecutionEnvironment beActiveDuring: [self spawnNewProcess]].
```

Comment:
I think that when Pavel did the commit, this change was already in his image and it did not show up in the Iceberg sync window. But it's just a guess :)